pages/_app.tsx

```diff
@@ -1,17 +1,20 @@
+import { ApolloProvider } from "@apollo/client";
 import { AppProps } from "next/app";
-import React from "react";
 import { ThemeProvider } from "styled-components";
+import client from "../apollo-client";
```

Comment:
used named import

Reply:
I tried this on my project and it didn't work for me. Curious to find out if it does work for you.